src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -749,16 +749,15 @@ private async Task ChangeOrganizationSponsorship(Organization org, OrganizationS
         prorationDate ??= DateTime.UtcNow;
```

Comment:
:pick: The parameter `prorationDate` is no longer being used, given the change to `subUpdateOptions`. If that's intended, feel free to remove it